### DISCUSSION TITLE
React-Konva with React Compiler

### DISCUSSION THREAD

**Discussion by devhe4d** [konvajs/konva#1779] [General]
Hey everyone,

I wonder if there is going to be any support for React compiler? I tried to add react-compiler to our project but React-Konva didn't accept that and throw so much errors ( I think because of module import/export issue )

**Comment by lavrton** [comment]
I hope it will be supported, I didn't look into that yet, but from the description of the compiler it should work just fine (as it is just some rewrites for better memo usage).

Did you try with `react-konva@19.0.0-0`?
